app/policies/pageflow/account_policy.rb

```diff
@@ -123,13 +124,13 @@ def destroy_membership_on?
     end
 
     def admin?
-      @user.admin?
+      user.admin?
     end
 
     def see_badge_belonging_to?
-      (@account.entries & @user.entries).any? ||
-        @user.memberships.on_accounts.as_previewer_or_above.where(entity: @account).any? ||
```

Comment:
Would this be okay in principle (not tested yet)? Would we want to catch the error/throw something else?
```
    def has_at_least_role?(role, on_as_many_accounts_as: 1)
      case on_as_many_accounts_as
      when 0
        does_not_have_role?(role)
      when 1
        has_role_at_least_once?(role)
      else
        if on_as_many_accounts_as > 1
          has_role_at_least?(role, on_as_many_accounts_as)
        else
          raise ArgumentError('Integer 0 or greater expected for '\
                              'on_as_many_accounts_as')
        end
      end
    end
```